inference/core/interfaces/stream/sinks.py

```diff
@@ -148,11 +157,15 @@ def _handle_frame_rendering(
         try:
             labels = [p["class"] for p in prediction["predictions"]]
             detections = sv.Detections.from_roboflow(prediction)
-            image = annotator.annotate(
-                scene=frame.image.copy(),
-                detections=detections,
-                labels=labels,
-            )
+            image = frame.image.copy()
+            for annotator in annotators:
+                kwargs = {
+                    "scene": image,
+                    "detections": detections,
+                }
+                if isinstance(annotator, sv.LabelAnnotator):
```

Comment:
[comment] example of why I believe callable is more versatile option.
Trade being made is not checking if something is callable vs adjusting for-loop with annotator chains to specifics of  `annotate(...)` invocations, not leaving options for customisation of other parameters

Reply:
Either way we end up with union of incompatible types, since we already had been using annotators from supervision it feels more natural to fix the original problem by allowing user to pass chain of annotators (BoxAnnotator is replaced by BoundingBoxAnnotator and LabelAnnotator)

Reply:
I still believe that better option is callable here, but it is also fine to leave as is